### PR TITLE
Enviroment variables for running tests locally

### PR DESCRIPTION
**Pull request by ioedeveloper**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+gist_token = 81b0b2c67a98fcca4827e2c8899db913829c7e04
```
Comment (yann300) [inline .env]: can we have something like
`gist_token = <token>`
then each will have to provide his own (that should be documented in the readme too).
Cause i can't commit that this token will always be working

File: test-browser/tests/gist.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 'use strict'
+require('dotenv').config()
```
Comment (yann300) [inline test-browser/tests/gist.js]: can this be added to `init.js` so we are sure `.env` is applied for all the tests, cause we might need env variable for other cases.

File: .env (hunk 1 above)
Comment (ioedeveloper) [reply .env]: Yes, okay. I'll make the change.

File: test-browser/tests/gist.js (hunk 2 above)
Comment (ioedeveloper) [reply test-browser/tests/gist.js]: Okay.

**Comment by yann300** [comment]
Thanks for the update, can the last commit title be changed to something like "move environment declaration to init phase & remove token", in order to describe more what is the change about.

**Comment by ioedeveloper** [comment]
> Thanks for the update, can the last commit title be changed to something like "move environment declaration to init phase & remove token", in order to describe more what is the change about.

I've renamed my commit title

**Pull request closed without merging**